build/amdclean.js

```diff
@@ -26,7 +26,7 @@ CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 */
 
-;(function(esprima, estraverse, escodegen, _) {
+;var sourcemapToAst;(function(esprima, estraverse, escodegen, _) {
```

Comment:
Could this be added to the IIFE like the other dependencies, instead of globally?